src/collective/frontpage/content/section.py

```diff
@@ -27,7 +22,15 @@ class ISection(model.Schema):
     background_color = schema.Choice(
         title=_(u'Background Color'),
         default='#0083BE',
-        vocabulary='collective.frontpage.SectionBackgroundColors',
+        vocabulary='collective.frontpage.SectionColors',
+        required=True,
+    )
+
+    primary_color = schema.Choice(
+        title=_(u'Primary Color'),
+        default='#0083BE',
+        description=_(u'Color for Icons and Buttons'),
+        vocabulary='collective.frontpage.SectionColors',
         required=True,
     )
 
```

Comment:
We should add more descriptions about the automatics (eg.: "The Background-Color will always be on top of the background-image. Make sure to use Transparency if you want to show the image.", but also for other fields in the schema!)